pydoof/search_api/search.py

```diff
@@ -96,6 +108,7 @@ def suggest(hashid: str, query: str, indices: List[str]=None, stats: bool=None,
             Note that [ and ] characters should be escaped (%5B and %5D) in all cases.
             Example: indices[]=product&indices[]=page; indices=products
         stats (boolean, optional): Enable/Disable this search in stats reports.
+            Default: true.
```

Comment:
```suggestion
        stats (boolean, optional): Enable/Disable this search in stats reports.
            Default: None.
```

None is the real default, but `Default: true.` line rather to be removed, because it says (boolean, optional)

Reply:
Yes but in our API is true by default, tha's the reason because I use this in all the Args. Is for our customers: "ey, remeber that this is default in our API".

Personally, I prefer to remove all this "docs" and [replace it for a URL to our docs](https://docs.doofinder.com/api/search/v5/index.html#operation/search_query), because that URL would be more updated that our python client.

Reply:
Oh, mmm, here is the documentation for the client, not for the api. If you put "None" as default (in the function argument), I expect it to be none when using it, at the docstring, etc.

I would prefer `stats=True` at the client to reflect the same as the api, that it has default true

Reply:
This is a bit difficult. For one part, I'm against replacing the docs for a docs URL. First, because as a developer, I wouldn't like to have to open a new tab every time I want to test something from a console, instead of a `?pydoof.search.suggest`. Second, because the python client could be behind the docs, and some arguments may not be available in a given version.

Now, the other part is what we are documenting here. We are writing the client documentation, so the real default value is `None`. The other part is what value the search API will give to that... And I wouldn't set a `stats=True` here as @sonic182 suggested because then we may be replacing the real Search API default value.

I think the best approach is leaving `Default: None`, and maybe, noting down that any `None` value won't be set in the request, and the API will use its default value.

Reply:
Roger that!

I changed rpp from 10 to None for the same reason. I will do the same in all args I see in the future in this client.